routes/game.rb

```diff
@@ -83,7 +83,7 @@ class Api
 
                 r.params['user'] = user.id
 
-                engine = engine.process_action(r.params)
+                engine = engine.process_action(r.params, add_autoactions: false)
```

Comment:
why is this false? isn't this the only place you'd want it to be true? maybe i don't understand what this param is doing

Reply:
The way it's designed is that only the client actually adds the autoactions and so the server won't then add them. This would cause inconsistency between the server and the client if we're mid-deploy.
I could allow the server to add them and hope the consistency remains the same, but I'm wary of there being races here